.github/scripts/bugzilla_reference_check.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+from __future__ import unicode_literals
+
+import argparse
```

Comment:
this is no more needed now, since you changed the way the PR number is passed to this script

Reply:
Good catch! Removed